ietf-optical-impairment-topology.yang

```diff
@@ -605,42 +635,316 @@ module ietf-optical-impairment-topology {
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The typ value correspond to typical case expected loss";        
+      }
+      leaf roadm-pmin {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the worst case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required."; 
+      }
+      leaf roadm-pmax {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the best case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required.";        
+      }
+      leaf roadm-ptyp {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the typical case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 

```

Comment:
I think the sentence 'To be noted ... is different for ptyp. in Colin slides the ptyp determines the power of the group of carriers, if several carriers are extrated on the same drop path (drop block)... not clear to me if it is for the pmaxxnb_carriers or Pminxnb_carriers ...  Maybe remove the sentence a say that a detailed description is for further work ?

Reply:
I've removed the sentence. We will check further with Colin.